### ISSUE TITLE
Glitch into bases using door

### ISSUE THREAD

**Issue by evermoreservers** [SmartlyDressedGames/Unturned-3.x-Community#555]
Easy way to glitch into any unclaimed base by putting doorframe next to it and just opening the door.
https://youtu.be/CznZIDdERWg

Sorry for bad record, i just got the video.

**Comment by termsgames** [comment]
Yeah theres been issues like this for a long time now. Unfortunately, nelsons failed to act on a lot of them. It could be to do with the limited support that unity offers with certain mechanics.
